packages/kbn-cell-actions/src/actions/filter/filter_out.ts

```diff
@@ -28,18 +37,26 @@ export const createFilterOutActionFactory = createCellActionFactory(
 
       return (
         data.length === 1 && // TODO Add support for multiple values
-        !!field.name
+        !!field.name &&
+        isTypeSupportedByDefaultActions(field.type as KBN_FIELD_TYPES)
```

Comment:
Since we are now assuming the type will always be `KBN_FIELD_TYPES`, maybe we should search for the places we do `type: column.type ?? 'keyword'`. _keyword_ type is no longer valid and it is confusing. 
Maybe we should set an empty string, to make sure the action won't be compatible.